casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -199,8 +197,13 @@ class TestNode[F[_]: Timer](
 
   def propagateBlock(deployDatums: Signed[DeployData]*)(nodes: TestNode[F]*): F[BlockMessage] =
     for {
-      block <- addBlock(deployDatums: _*)
-      _ <- nodes.toList.traverse_ { node =>
+      _       <- Log[F].debug((s"\n$name creating block"))
```

Comment:
this is just optimisation I did when debugging one test issue - now it node does not propagate to itself an prints some useful logs